fullstop/cli.py

```diff
@@ -235,14 +235,23 @@ def list_violations(config, output, since, region, meta, remeta, limit, all, **k
     rows.reverse()
 
     with OutputFormat(output):
-        print_table(['account_id', 'region', 'violation_type', 'instance_id', 'meta_info', 'comment', 'created_time'],
+        print_table(['account_id',
+                     'region',
+                     'id',
+                     'violation_type',
+                     'instance_id',
+                     'meta_info',
+                     'comment',
+                     'created_time'],
                     rows, titles={'created_time': 'Created'})
 
 
 @cli.command('resolve-violations')
 @click.option('--accounts', metavar='ACCOUNT_IDS',
               help='AWS account IDs to filter for (default: your configured accounts)')
 @click.option('-s', '--since', default='1d', metavar='TIME_SPEC', help='Only show violations newer than')
+@click.option('-i', '--violation_ids', default='', metavar='VIOLATION_ID', help='resolve this specific violations, ' +
```

Comment:
please use dash in the option, i.e. `--violation-ids` (the function automatically uses underscore)